projects/canopy/src/lib/forms/sort-code/sort-code.component.ts

```diff
@@ -40,6 +40,10 @@ let nextUniqueId = 0;
   styleUrls: ['./sort-code.component.scss'],
   encapsulation: ViewEncapsulation.None,
 })
+/**
+ * @deprecated LgSortCodeComponent is deprecated.
```

Comment:
Could you add some details about this to https://github.com/Legal-and-General/canopy/issues/315 please?

Reply:
Done 👍